pkg/chunked/cache_linux.go

```diff
@@ -488,26 +626,41 @@ func readCacheFileFromMemory(bigDataBuffer []byte) (*cacheFile, error) {
 	if err := binary.Read(bigData, binary.LittleEndian, &digestLen); err != nil {
 		return nil, err
 	}
+
+	bloomFilter, err := readBloomFilter(bigData)
```

Comment:
What is a typical size of the filter? Would it pay off to access the mapped data directly instead of making a copy?

(I didn’t even try to estimate, for all I know this might clearly not be worth the extra complexity. If it is a few percent of the total size, let’s not bother.)